### DISCUSSION TITLE
Scenarios and Professions

### DISCUSSION THREAD

**Discussion by Relgar** [cataclysmbn/Cataclysm-BN#2173] [General]
So this was based off an earlier idea I had in Discord. I'm a big believer in different starts in any RPG type games (Skyrim, Fallout, Nethack, Dwarf Fortress, etc.) as I feel it allows for more replayability and approaching the game at a different angle with different challenges. As such, I feel that professions should be tied in one of three ways:

1. Submenus - the 'all professions' scenarios are cluttered beyond belief and throw way too many professions at the player, some of which have little meaningful differences in terms of gameplay. Having professions under a submenu (i.e. Military would have all military professions, Construction would have specific professions related to building etc.)  may make this divide easier to navigate and to group like-minded professions together. Unfortunately the work required for this probably isn't trivial.

2. Scenario-constrained Professions - pull all professions each into a specific scenario and remove the idea of 'all-professions available' scenarios altogether. This creates specific starts that correlate both thematically (and mechanically) with the scenario, and also become easier to balance as the profession is balanced relative to the starting scenario instead of every other profession. This can also set the scene for immediate start missions with NPCs, which can offer more flavour.

3. A combination of both - i.e. In an Urban Scenario, you have a sub-scenario of a Hospital (as opposed to a location), where you can play a Patient, a Doctor, a Medic, a Surgeon, etc. Alternatively, you can pick a sub-scenario of an Office Tower, where you could be a Janitor, or Office Worker, or Security Guard. Contrast this with a Wilderness Scenario where you could play a sub-scenario of a Camping Expedition, which gives you access to Survivalist, Camper, Prepper, etc.

Let me know if you think this is a good idea or if there's a better way to streamline the huge amount of professions we have.

**Comment by silverwing235** [comment]
So, IOW, things like 'Survivor' per se, would be too generic to fit if I understand you correctly?

About the other matter, # 3 gets my vote.

**Reply by Relgar** [reply]
Yeah, survivor kind of gives the impression you're someone who knows how to live off the land etc. so I'd group them more towards the camping aspect of things.

**Comment by PhilosopherZ** [comment]
你好，我已收到你的邮件。
